arb_os/arbowner.mini

```diff
@@ -75,6 +101,144 @@ impure func arbowner_giveOwnership(topFrame: EvmCallFrame, calldata: ByteArray)
     evmOp_return(0, 0);
 }
 
+impure func arbowner_addToReserveFunds(topFrame: EvmCallFrame, calldata: ByteArray) { // () payable
```

Comment:
Is there a reason this is limited to the owner? There might be advantages to making this publicly callable

Reply:
It could be publicly callable, in principle.  For safety, I made everything in ArbOwner callable only by the owner, so putting this here makes it owner-only. We could move it out, but then the getter and setter would be in separate contract, which isn't great.